go/libraries/doltcore/sqle/tables.go

```diff
@@ -96,6 +98,113 @@ type DoltTable struct {
 	overriddenSchema schema.Schema
 }
 
+func (t *DoltTable) SkipIndexCosting() bool {
+	return false
+}
+
+func (t *DoltTable) LookupForExpressions(ctx *sql.Context, exprs ...sql.Expression) (sql.IndexLookup, *sql.FuncDepSet, sql.Expression, bool, error) {
+	root, err := t.workingRoot(ctx)
+	if err != nil {
+		return sql.IndexLookup{}, nil, nil, false, err
+	}
+
+	schHash, err := root.GetTableSchemaHash(ctx, doltdb.TableName{Name: t.tableName}, t.overriddenSchema != nil)
```

Comment:
This isn't going to work for Doltgres heads up

You might want to add a TableName() method to DoltTable, you can get the schema name from the db.

Reply:
Hmm, if true there are tons of places where we ignore the schema in `sqle/tables.go`. Will patch up the ones I see.